text/0039-sdk-performance-issue-creation.md

```diff
@@ -0,0 +1,72 @@
+* Start Date: 2022-11-28
+* RFC Type: decision
+* RFC PR: [#33](https://github.com/getsentry/rfcs/pull/33)
+* RFC Status: draft
+* RFC Driver: [Philipp Hofmann](https://github.com/philipphofmann)
+
+# Summary
+
+This RFC aims to clarify which types of performance issues SDKs shall create.
```

Comment:
Does this need to be a generic RFC? I was thinking we might focus on this case (file I/O on the main thread) in particular, and make a decision about that one alone.

I don't think trying to come up with generic performance issue implementation strategies is that meaningful. We've been asked to start from root causes, and look at the best possible detection strategy for each one individually. Trying to delimit "ingest detectors" vs "SDK detectors" is looking at the problem backwards IMO.

Reply:
Good point, thanks 🙏 .